assets/config/assets.onj

```diff
@@ -194,6 +194,10 @@ textures: [
         name: "map_background_wasteland_tileable",
         file: "textures/map/backgrounds/wasteland.png",
     },
+    {
+        name: "map_background_bewitchedForest_tileable",
```

Comment:
why is the biome name not in snake_case?

Reply:
since i think it doesn't make sense (ik we should, and if you want to, i can change it, however i think it is better readable that way, even if we are mixing 2 different types of writing)